index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/9fa2fe115b.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header id="header" class="p-3">
+      <nav id="nav-bar" class="fixed-top">
+          <img
+              id="header-img"
+              src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+              alt="original trombones logo"
+          />
+
+          <ul>
+              <li><a class="nav-link text-decoration-none link-dark" href="#features">Features</a></li>
+              <li><a class="nav-link text-decoration-none link-dark" href="#how-it-works">How it Works</a></li>
+              <li><a class="nav-link text-decoration-none link-dark" href="#pricing">Pricing</a></li>
+          </ul>
+      </nav>
+  </header>
+  <div class="container">
+      <div class="row my-5">
+          <div class="col text-center my-3">
+              <form id="form" name="form" method="get" action="https://www.freecodecamp.com/email-submit">
+                  <h4 class="fw-bold">Handcrafted, home-made masterpieces</h4>
+                  <input id="email" name="email" type="email" placeholder="Enter you email address" class="form-control mx-auto mt-3" style="width:250px" required />
```

Comment:
Do not recommend using inline style, please only use external CSS

Reply:
- [ ] I think it would be better to give a reason why the person should use only external CSS👍.

Reply:
Yes, please. Send the pull request to the dashboard for a re-review🏂